visualization.shadertoy/resources/language/resource.language.en_gb/strings.po

```diff
@@ -123,3 +123,39 @@ msgstr ""
 msgctxt "#30029"
 msgid "Channel 3 as *.png"
 msgstr ""
+
+msgctxt "#30030"
+msgid "Shadertoy Visualization"
+msgstr ""
+
+msgctxt "#30031"
+msgid "Incorrect reading of the given JSON file, check whether it is correct and available."
```

Comment:
Cannot read the JSON file. Please check the formatting is correct and the file is available.